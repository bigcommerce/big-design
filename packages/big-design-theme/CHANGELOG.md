# Change Log

## 1.3.0

### Minor Changes

- 042d470: Change the documentation and code to use Source Sans 3 instead of Source Sans Pro. Source Sans Pro is licensed by Adobe now which means we need to the utilize the open license with Source Sans 3.

## 1.2.0

### Minor Changes

- e323932: added negative values to spacing definitions

## 1.1.0

### Minor Changes

- f8665f3: Added wide breakpoint to theme definition to best adapt responsive features to wide screens

All notable changes to this project will be documented in this file.
See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.

## [1.0.2](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@1.0.1...@bigcommerce/big-design-theme@1.0.2) (2024-08-06)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [1.0.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@1.0.0...@bigcommerce/big-design-theme@1.0.1) (2024-06-11)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [1.0.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.21.1...@bigcommerce/big-design-theme@1.0.0) (2024-04-17)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.21.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.21.0...@bigcommerce/big-design-theme@0.21.1) (2024-04-09)

### Bug Fixes

- typescript build output with wrong imports ([a812497](https://github.com/bigcommerce/big-design/commit/a81249798e20c10754c938b2b3d3dcc95d81aa0c))

# [0.21.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.20.1...@bigcommerce/big-design-theme@0.21.0) (2024-04-02)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.20.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.20.0...@bigcommerce/big-design-theme@0.20.1) (2024-01-23)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.20.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.19.2...@bigcommerce/big-design-theme@0.20.0) (2023-10-09)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.19.2](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.19.1...@bigcommerce/big-design-theme@0.19.2) (2023-06-14)

### Bug Fixes

- declaration tsconfig for theme publishing ([4789b8b](https://github.com/bigcommerce/big-design/commit/4789b8b35d3fbfbd42d0312de66cd1b48bef0f39))

## [0.19.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.19.0...@bigcommerce/big-design-theme@0.19.1) (2023-06-09)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.19.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.18.0...@bigcommerce/big-design-theme@0.19.0) (2023-04-05)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.18.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.17.0...@bigcommerce/big-design-theme@0.18.0) (2023-01-09)

### Features

- convert to react@18 only ([#1044](https://github.com/bigcommerce/big-design/issues/1044)) ([06c4697](https://github.com/bigcommerce/big-design/commit/06c469721bc06d1f872bed6bf5a46e6b568644e5))

### BREAKING CHANGES

- Requires `react@18` or higher.

# [0.17.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.16.1...@bigcommerce/big-design-theme@0.17.0) (2022-09-27)

### Features

- **deps:** react@18 support ([51c688b](https://github.com/bigcommerce/big-design/commit/51c688b282277964eb01c1ed67c5c27cc0bc4c3e))

## [0.16.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.16.0...@bigcommerce/big-design-theme@0.16.1) (2022-09-27)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.16.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.15.1-alpha.0...@bigcommerce/big-design-theme@0.16.0) (2022-08-17)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.15.1-alpha.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.15.0...@bigcommerce/big-design-theme@0.15.1-alpha.0) (2022-08-10)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.15.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.14.1...@bigcommerce/big-design-theme@0.15.0) (2022-07-06)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.14.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.14.0...@bigcommerce/big-design-theme@0.14.1) (2022-06-10)

### Features

- **theme:** re-export hideVisually from polished ([e102e6d](https://github.com/bigcommerce/big-design/commit/e102e6dfc6d28b311556946d3e90d2323885713f))

# [0.14.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.2-alpha.1...@bigcommerce/big-design-theme@0.14.0) (2022-05-11)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.13.2-alpha.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.2-alpha.0...@bigcommerce/big-design-theme@0.13.2-alpha.1) (2022-03-22)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.13.2-alpha.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.1...@bigcommerce/big-design-theme@0.13.2-alpha.0) (2021-10-28)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.13.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.0...@bigcommerce/big-design-theme@0.13.1) (2021-09-03)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.13.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.0-alpha.2...@bigcommerce/big-design-theme@0.13.0) (2021-08-04)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.13.0-alpha.2](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.0-alpha.1...@bigcommerce/big-design-theme@0.13.0-alpha.2) (2021-08-02)

### chore

- **deps:** upgrade to styled-components v5 ([#566](https://github.com/bigcommerce/big-design/issues/566)) ([dd83711](https://github.com/bigcommerce/big-design/commit/dd83711797eb8aaa7a8406eebf1383116bff2420))

### Features

- **theme:** add listReset helper ([#570](https://github.com/bigcommerce/big-design/issues/570)) ([385b142](https://github.com/bigcommerce/big-design/commit/385b1426ee39171343070f9b31ad4e31d834b272))

### BREAKING CHANGES

- **deps:** You will need to update to styled-components v5. In addition,
  you will need to import the base fonts in your <head> element.
  See the "Getting Started" page or README.md for an example.

# [0.13.0-alpha.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.13.0-alpha.0...@bigcommerce/big-design-theme@0.13.0-alpha.1) (2021-07-06)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.13.0-alpha.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.12.1...@bigcommerce/big-design-theme@0.13.0-alpha.0) (2021-06-10)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.12.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.12.0...@bigcommerce/big-design-theme@0.12.1) (2021-03-29)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.12.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.11.0...@bigcommerce/big-design-theme@0.12.0) (2021-03-08)

### Bug Fixes

- **docs:** Specify styled-components version in the README ([#490](https://github.com/bigcommerce/big-design/issues/490)) ([c92403e](https://github.com/bigcommerce/big-design/commit/c92403edf8e29bf1afd3c9c163c37be6ffe17a6c))

### Features

- **theme:** add currentColor, inherit, and transparent props to colors ([#504](https://github.com/bigcommerce/big-design/issues/504)) ([3ef9b18](https://github.com/bigcommerce/big-design/commit/3ef9b1899ccef4c282f2f293e8fdd34d204e0420))

# [0.11.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.10.1...@bigcommerce/big-design-theme@0.11.0) (2020-11-12)

### Bug Fixes

- webpack 5 compatibility ([#466](https://github.com/bigcommerce/big-design/issues/466)) ([7e66238](https://github.com/bigcommerce/big-design/commit/7e66238ca42cb27d91b1a80cc9e1c8014808e27b))
- **component:** updates progress circle to use em instead of rem due to error in Safari 14 ([#472](https://github.com/bigcommerce/big-design/issues/472)) ([3bedcc1](https://github.com/bigcommerce/big-design/commit/3bedcc1be8e5f5a8c9c63d634a1c567dd3016002))

## [0.10.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.10.0...@bigcommerce/big-design-theme@0.10.1) (2020-08-31)

### Features

- **theme:** add DefaultTheme extension types ([cdfeb49](https://github.com/bigcommerce/big-design/commit/cdfeb493973cce48ed92e30951be2b8bc87ed194))

# [0.10.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.9.0...@bigcommerce/big-design-theme@0.10.0) (2020-06-16)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.9.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.8.0...@bigcommerce/big-design-theme@0.9.0) (2020-04-29)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.8.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.7.0...@bigcommerce/big-design-theme@0.8.0) (2020-04-20)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.7.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.6.0...@bigcommerce/big-design-theme@0.7.0) (2020-03-25)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# [0.6.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.5.0...@bigcommerce/big-design-theme@0.6.0) (2020-02-21)

### Features

- **component:** add Alerts, InlineAlerts, and Message components ([#340](https://github.com/bigcommerce/big-design/issues/340)) ([d541276](https://github.com/bigcommerce/big-design/commit/d54127603fba47b46cb35c3db4caf53ab24bafc3))

# [0.5.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.4.0...@bigcommerce/big-design-theme@0.5.0) (2020-02-06)

### Bug Fixes

- add react-hooks eslint and fix missing deps/issues ([#330](https://github.com/bigcommerce/big-design/issues/330)) ([da3fbd6](https://github.com/bigcommerce/big-design/commit/da3fbd68181e98e43a95de7fce9956be91afc9b8))

# [0.4.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.3.1...@bigcommerce/big-design-theme@0.4.0) (2019-12-09)

### Features

- **component:** add tab trap for modals and dialogs ([dcacbf9](https://github.com/bigcommerce/big-design/commit/dcacbf96a38bef1134e2a8dcbd986f6362e0e2b7))

## [0.3.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.3.0...@bigcommerce/big-design-theme@0.3.1) (2019-11-12)

### Features

- **all:** better tree shaking ([c0998a7](https://github.com/bigcommerce/big-design/commit/c0998a7))

# [0.3.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.2.0...@bigcommerce/big-design-theme@0.3.0) (2019-11-11)

### Features

- **all:** bump styled-components peerDependency version ([fd89fa3](https://github.com/bigcommerce/big-design/commit/fd89fa3))
- **component:** tooltip for dropdown item ([#228](https://github.com/bigcommerce/big-design/issues/228)) ([4e5fc50](https://github.com/bigcommerce/big-design/commit/4e5fc50))

### BREAKING CHANGES

- **all:** bumped peer dependency of styled-components to `^4.3.0`

# [0.2.0](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.1.2...@bigcommerce/big-design-theme@0.2.0) (2019-10-29)

**Note:** Version bump only for package @bigcommerce/big-design-theme

## [0.1.2](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.1.1...@bigcommerce/big-design-theme@0.1.2) (2019-08-19)

### Bug Fixes

- **component:** add missing z-index to components ([#176](https://github.com/bigcommerce/big-design/issues/176)) ([800d2fc](https://github.com/bigcommerce/big-design/commit/800d2fc))

### BREAKING CHANGES

- **component:** `theme.zIndex` no longer has `theme.zIndex.dropdown`

## [0.1.1](https://github.com/bigcommerce/big-design/compare/@bigcommerce/big-design-theme@0.1.0...@bigcommerce/big-design-theme@0.1.1) (2019-08-15)

**Note:** Version bump only for package @bigcommerce/big-design-theme

# 0.1.0 (2019-08-13)

### Bug Fixes

- **theme:** export missing properties ([f439df0](https://github.com/bigcommerce/big-design/commit/f439df0))
- **theme:** issue with font sizes in createTheme ([#135](https://github.com/bigcommerce/big-design/issues/135)) ([313ff46](https://github.com/bigcommerce/big-design/commit/313ff46))

### Features

- **theme:** initial package setup ([f734dfb](https://github.com/bigcommerce/big-design/commit/f734dfb))
- **theme:** rename elevation to shadow ([659c6ad](https://github.com/bigcommerce/big-design/commit/659c6ad))

### BREAKING CHANGES

- **theme:** `elevation` is now called `shadow`
