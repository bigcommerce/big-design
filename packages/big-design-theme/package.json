{
  "name": "@bigcommerce/big-design-theme",
  "version": "1.1.0",
  "sideEffects": false,
  "main": "src/index.ts",
  "license": "(MIT AND CC-BY-3.0)",
  "author": "BigCommerce Inc",
  "repository": {
    "type": "git",
    "url": "https://github.com/bigcommerce/big-design.git",
    "directory": "packages/big-design-theme"
  },
  "scripts": {
    "build": "pnpm run build:cjs && pnpm run build:es && pnpm run build:dt",
    "build:cjs": "NODE_ENV=production BABEL_ENV=cjs babel --extensions \".ts,.tsx\" ./src --out-dir ./dist/cjs",
    "build:es": "NODE_ENV=production BABEL_ENV=es babel --extensions \".ts,.tsx\" ./src --out-dir ./dist/es",
    "build:dt": "tsc -p tsconfig.declarations.json --emitDeclarationOnly",
    "lint": "eslint . --max-warnings 0",
    "prepack": "big-design-prepack",
    "postpack": "big-design-postpack",
    "test": "jest",
    "test:watch": "pnpm run test --watch",
    "typecheck": "tsc --noEmit"
  },
  "files": [
    "dist"
  ],
  "publishConfig": {
    "access": "public",
    "main": "dist/cjs/index.js",
    "module": "dist/es/index.js",
    "typings": "dist/index.d.ts"
  },
  "prettier": "@bigcommerce/eslint-config/prettier",
  "dependencies": {
    "@babel/runtime": "^7.26.0",
    "polished": "^4.3.1"
  },
  "peerDependencies": {
    "react": "^18.0.0",
    "react-dom": "^18.0.0",
    "styled-components": "^5.3.5"
  },
  "devDependencies": {
    "@babel/cli": "^7.25.7",
    "@babel/core": "^7.26.0",
    "@babel/plugin-transform-runtime": "^7.25.9",
    "@babel/preset-env": "^7.26.0",
    "@babel/preset-react": "^7.26.3",
    "@babel/preset-typescript": "^7.26.0",
    "@bigcommerce/configs": "workspace:^",
    "@bigcommerce/pack": "workspace:^",
    "@styled/typescript-styled-plugin": "^1.0.1",
    "@swc/core": "^1.10.4",
    "@swc/jest": "^0.2.37",
    "@swc/plugin-styled-components": "^6.0.2",
    "@types/jest": "^29.5.14",
    "@types/styled-components": "^5.1.34",
    "babel-jest": "^29.0.2",
    "babel-plugin-styled-components": "^2.0.7",
    "jest": "^29.7.0",
    "jest-environment-jsdom": "^29.4.1",
    "jest-fail-on-console": "^3.3.1",
    "jest-styled-components": "^7.1.1",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "styled-components": "^5.3.11",
    "typescript": "^5.6.3"
  }
}
