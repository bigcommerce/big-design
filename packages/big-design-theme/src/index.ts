import { createHelpers, Helpers } from './helpers';
import { ThemeOptions, themeOptions } from './options';
import { Border, BorderRadius, createBorder, createBorderRadius } from './system/border';
import { Breakpoints, breakpoints, BreakpointValues, breakpointValues } from './system/breakpoints';
import { Colors, colors } from './system/colors';
import * as keyframes from './system/keyframes';
import { createLineHeight, LineHeight } from './system/line-height';
import { Shadow, shadow } from './system/shadow';
import { createSpacing, Spacing } from './system/spacing';
import { createTypography, Typography } from './system/typography';
import { ZIndex, zIndex } from './system/z-index';

export * from './helpers';
export * from './system';

export interface ThemeInterface {
  border: Border;
  borderRadius: BorderRadius;
  breakpointValues: BreakpointValues;
  breakpoints: Breakpoints;
  colors: Colors;
  helpers: Helpers;
  keyframes: typeof keyframes;
  lineHeight: LineHeight;
  shadow: Shadow;
  spacing: Spacing;
  typography: Typography;
  zIndex: ZIndex;
}

export const createTheme = (customOptions: Partial<ThemeOptions> = {}): ThemeInterface => {
  themeOptions.setOptions(customOptions);

  return {
    border: createBorder(),
    borderRadius: createBorderRadius(),
    breakpointValues,
    breakpoints,
    colors,
    helpers: createHelpers(),
    keyframes,
    lineHeight: createLineHeight(),
    shadow,
    spacing: createSpacing(),
    typography: createTypography(),
    zIndex,
  };
};

export const theme: ThemeInterface = createTheme();
