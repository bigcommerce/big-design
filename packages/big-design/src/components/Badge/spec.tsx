import { theme } from '@bigcommerce/big-design-theme';
import { render, screen } from '@testing-library/react';
import React from 'react';
import 'jest-styled-components';

import { Badge } from './index';

test('has margin props', () => {
  const { rerender } = render(<Badge label="Badge" />);
  const badge = screen.getByText('Badge');

  expect(badge).not.toHaveStyle('margin: 1rem');

  rerender(<Badge label="Badge" margin="medium" />);

  expect(badge).toHaveStyle('margin: 1rem');
});

test('render default Badge', () => {
  render(<Badge label="Badge" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.secondary60}`);
});

test('render success Badge', () => {
  render(<Badge label="Badge" variant="success" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.success50}`);
});

test('render danger Badge', () => {
  render(<Badge label="Badge" variant="danger" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.danger40}`);
});

test('render warning Badge', () => {
  render(<Badge label="Badge" variant="warning" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.warning40}`);
});

test('render secondary Badge', () => {
  render(<Badge label="Badge" variant="secondary" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.secondary60}`);
});

test('render primary Badge', () => {
  render(<Badge label="Badge" variant="primary" />);

  const badge = screen.getByText('Badge');

  expect(badge).toMatchSnapshot();
  expect(badge).toHaveStyle(`background-color: ${theme.colors.primary40}`);
});

test("doesn't render if label prop is invalid", () => {
  // @ts-expect-error ignoring since label={Component} is not a valid prop
  const { container } = render(<Badge label={<p>Label</p>} />);

  expect(container).toBeEmptyDOMElement();
});
