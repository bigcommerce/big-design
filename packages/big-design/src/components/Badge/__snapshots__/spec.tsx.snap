// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`render danger Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  background-color: #DB3643;
}

<span
  class="c0"
>
  Badge
</span>
`;

exports[`render default Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  background-color: #5E637A;
}

<span
  class="c0"
>
  Badge
</span>
`;

exports[`render primary Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  background-color: #3C64F4;
}

<span
  class="c0"
>
  Badge
</span>
`;

exports[`render secondary Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  background-color: #5E637A;
}

<span
  class="c0"
>
  Badge
</span>
`;

exports[`render success Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  background-color: #208831;
}

<span
  class="c0"
>
  Badge
</span>
`;

exports[`render warning Badge 1`] = `
.c0 {
  color: #FFFFFF;
  border-radius: 0.25rem;
  display: inline-block;
  font-size: 0.75rem;
  font-weight: 600;
  line-height: 1.25rem;
  text-align: center;
  text-transform: uppercase;
  vertical-align: middle;
  padding: 0 0.5rem;
  color: #313440;
  background-color: #FFBF00;
}

<span
  class="c0"
>
  Badge
</span>
`;
